.docker/Dockerfile.meteor

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:focal
+
+ARG RELEASE
+
+ADD https://s3.amazonaws.com/com.meteor.static/packages-bootstrap/$RELEASE/meteor-bootstrap-os.linux.x86_64.tar.gz /
```

Comment:
Is there a non-s3 path we could link to?  I know if it was our s3 I wouldn't want people linking direct to it like this.  Can be expensive.  AWS likes to get you on bandwidth

Reply:
I took this directly from the install script - https://install.meteor.com - line no. 133.

Since their script is using it I figured we could too. Not sure if we can use some other endpoint redirect in this case.

Reply:
JFYI - the new `npm i -g meteor` doesn't work for meteor@2.2 which we need (`npm i -g meteor@2.3.1` see https://www.npmjs.com/package/meteor for version correlation between the npm package and meteor).

Reply:
Ok - scratch that - found one https://github.com/meteor/meteor/blob/290bf55e3c8c002ff35c0f9553383d7a2f9f4c2e/npm-packages/meteor-installer/install.js#L43

https://packages.meteor.com/bootstrap-link?arch=os.linux.x86_64&release=2.2 

This should work :)